src/app/lib/utilities/ichefURL/index.js

```diff
@@ -2,28 +2,27 @@
 const WEBP_ORIGIN_CODES = ['cpsdevpb', 'cpsprodpb'];
 
 const buildPlaceholderSrc = (src, resolution) => {
-  const parts = src.split('/');
-  const [domain, media, imgService, width, ...extraParts] = parts;
-  const definedWidth = width.replace('$width', resolution);
-  const domainWithProtocol = `https://${domain}`;
-
+  // return urn schemes unmodified
+  if (src.includes('urn:')) return src;
+  const noProtocolSrc = src.replace('https://', '');
+  const urlParts = noProtocolSrc.split('/');
+  const [domain, mediaType, imgService, ...extraParts] = urlParts;
+  // Slice use to cut off the old resolution
+  const newExtraParts = extraParts.slice(1);
+  const newResolution = `${resolution}xn`;
   const newUrl = [
-    domainWithProtocol,
-    media,
+    'https:/',
```

Comment:
Should this be 'https://' rather than 'https:/'?

Reply:
Notice how in line 21 we use `return newUrl.join('/');` This will https:/ and the domain with a `/` leaving us with https://....

Reply:
I thinkl the 1 slash http is a little bit eye catching and weird. a future dev might think it's a bug and sink some time into looking at it

What about
```
return 'http://'.concat(newUrl.join('/'));
```